doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -141,3 +141,11 @@ By default this Kconfig option is set to 1 second.
 .. note::
   The :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is set to ``y`` by default.
   To disable removing application-specific non-volatile data when the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` Kconfig option is selected, set the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option to ``n``.
+
+Read Client functionality
+=========================
+
+Read Client functionality needs to be enabled if the device needs to read attributes from another device in the Matter network.
+To enable it set the :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` to ``y`` and to ``n`` to disable.
+In Matter samples we have disabled this functionality by default for all samples except :ref:`matter_light_switch_sample`, and :ref:`matter_bridge_app` because both of them need to read attributes from the bound devices.
```

Comment:
```suggestion
This functionality is disabled by default for Matter samples in the |NCS|, except for ones that need to read attributes from the bound devices, such as the :ref:`matter_light_switch_sample` sample and the :ref:`matter_bridge_app` application.
```
Agree that having an inclusive list is not a good idea, so making the current ones into examples solves the need to update this line unless the functionality would be disabled by default for either of the mentioned ones.